pkgs/desktops/gnome-3/core/evince/default.nix

```diff
@@ -67,12 +85,29 @@ stdenv.mkDerivation rec {
     librsvg
     adwaita-icon-theme
     gspell
-  ] ++ stdenv.lib.optional supportXPS libgxps;
+    dbus # only needed to find the service directory
+    texlive.bin.core # kpathsea for DVI support
+    t1lib
+  ] ++ stdenv.lib.optional supportXPS libgxps
+    ++ stdenv.lib.optionals supportMultimedia (with gst_all_1; [
+      gstreamer gst-plugins-base gst-plugins-good gst-plugins-bad gst-plugins-ugly gst-libav ]);
 
-  configureFlags = [
-    "--disable-nautilus" # Do not build nautilus plugin
-    "--enable-introspection"
-    (if supportXPS then "--enable-xps" else "--disable-xps")
+  mesonFlags = [
+    "-Dnautilus=false"
+    "-Dps=enabled"
+    "-Dcomics=enabled"
+    "-Ddjvu=enabled"
+    "-Dpdf=enabled"
+    "-Dps=enabled"
+    "-Dtiff=enabled"
+    "-Ddvi=enabled"
+    "-Dt1lib=enabled"
+    "-Dxps=${if supportXPS then "enabled" else "disabled"}"
+    "-Dmultimedia=${if supportMultimedia then "enabled" else "disabled"}"
+    "-Dgtk_unix_print=enabled"
```

Comment:
Can we use `-Dauto_features=enabled`?

Reply:
Yes, this looks much nicer.  Why are auto features even disabled by default?

Reply:
I did not really think it through when I added it :stuck_out_tongue_winking_eye:

Reply:
Perhaps due to less predictability in case some packages propagate others?